manifests/database.pp

```diff
@@ -72,6 +72,8 @@
 #   The default collation of the database.
 #   default: utf8_general_ci
 #
+# [* database_tablespace*]
```

Comment:
Can you please clarify which database type this affects? It would be also nice to have the code failing if the parameter would be set for an other database type as postgresql

Reply:
Done!